packages/analytics-js/src/components/utilities/url.ts

```diff
@@ -1,19 +1,20 @@
+import { UTMParameters } from '@rudderstack/analytics-js/state/types';
 import { Nullable } from '@rudderstack/analytics-js/types';
 
 /**
  * Removes trailing slash from url
  * @param url
  * @returns url
  */
-const removeTrailingSlashes = (url: string | null): Nullable<string> =>
+export const removeTrailingSlashes = (url: string | null): Nullable<string> =>
```

Comment:
please put exports at the bottom of the files

Reply:
Addressed.